### PR TITLE
Containerized Docker Application Lifecycle - 04.13

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary
Guide <-> e-book syncing - Chapter 4 - Article 13

### CODE REVIEW

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 1]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
 ![Option dialog to select Target OS for Dockerfile.](media/select-linux-docker-support.png)
 
-**Figure 4-39**. Selecting Linux containers.
+**Figure 4-39**. Selecting Linux Containers.
 
-With these simple steps, you will have your ASP.NET Core 2.1 application running on a Linux container.
+With these simple steps, you will have your ASP.NET Core 2.1 application running on a Linux Container.
 
-As you can see, the integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
+As you can see the, integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
As you can see, the integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 2]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
 
-Now you can press **F5** to build and run your application.
+Now you can run your application with the key **F5** or using the Play Button
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
Now you can run your application with the **F5** key or by using the Play Button.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 3]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
 
-After running the project, you can list the images using the `docker images` command. You should see the `mssampleapplication` image created with the automatic deploy of our project with Visual Studio 2017.
+After running the project, you can list the images using the `docker images` command, and you should see the `mssampleapplication` image created with the automatic deploy of our project with Visual Studio 2017.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
After running the project, you can list the images using the `docker images` command, and you should see the `mssampleapplication` image created with the automatic deployment of our project with Visual Studio 2017.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 4]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
-![Console output from the docker images command, shows a list with: Repository, Tag, Image ID, Created (date), and Size.](media/docker-images-command.png)
+![Console output from the docker images command, shows a list with: Repository, Tag, Image Id, Created (date), and Size.](media/docker-images-command.png)
 
-**Figure 4-40**. View of Docker images
+**Figure 4-40**. View of docker images
 
 ## Register the Solution in the Azure Container Registry
 
-Upload the image to any Docker registry, like [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/) or Docker Hub so the images can be deployed to the AKS cluster from that registry. In this case, we’re uploading the image to Azure Container Registry.
+Now, we have to upload the image to any Docker registry, like [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/) or Docker Hub so the images can be deployed to the AKS cluster from that registry. In this case, we’re uploading the image to Azure Container Registry.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
Now, we have to upload the image to any Docker registry, like [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/) or Docker Hub, so the images can be deployed to the AKS cluster from that registry. In this case, we’re uploading the image to Azure Container Registry.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 5]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
 
 ### Create the image in Release mode
 
-Create the image in **Release** Mode (ready for production) changing to Release as shown here and press F5 to run the application again.
+We'll now create the image in **Release** Mode (ready for production) changing to Release as shown here and running the application as did before.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
We'll now create the image in **Release** mode (ready for production) by changing to **Release** as in Figure 4-41 and running the application as did we before.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 6]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
 
 ![Toolbar option in VS to build in release mode.](media/select-release-mode.png)
 
 **Figure 4-41**. Selecting Release Mode
 
-If you execute the `docker image` command, you'll see both images created. One for `debug` and the other for `release` mode.
+If you execute the `docker image` command, you'll see both images created one for `debug` and the other for `release` mode.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
If you execute the `docker image` command, you'll see both images created, one for `debug` and the other for `release` mode.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 7]
```diff
@@ -103,31 +97,31 @@ az acr list --resource-group MSSampleResourceGroup --query "[].{acrLoginServer:l
 
 **Figure 4-43**. Get the name of the registry using PowerShell
 
-In both cases, you'll obtain the name. In our example, `mssampleacr.azurecr.io`.
+In both cases, you'll obtain the name, in our example `mssampleacr.azurecr.io`
 
-Now you can tag the image, taking the latest image (Release image), with the following command:
+Now you can Tag the image, taking the latest image (Release image), with the command:
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
Now you can tag the image, taking the latest image (the Release image), with the command:
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 8]
```diff
@@ -103,31 +97,31 @@ az acr list --resource-group MSSampleResourceGroup --query "[].{acrLoginServer:l
...
 
 ```console
 docker tag mssampleaksapplication:latest mssampleacr.azurecr.io/mssampleaksapplication:v1
 ```
 
-After running the `docker tag` command, list the images with the `docker images` command. You should see the image with the new tag.
+After running the `docker tag` command, list the images with the `docker images` command, you should see the image with the new tag.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
After running the `docker tag` command, list the images with the `docker images` command, and you should see the image with the new tag.
```

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 9]
```diff
@@ -137,42 +131,12 @@ You can see below the result you should get when the process completes:
...
-spec:
-  ports:
-    - name: http-port
-      port: 80
-      targetPort: 80
-  selector:
-    app: mssample-kub-app
-  type: LoadBalancer
-```
+The next step is to deploy your container into your AKS Kubernetes cluster, for that you need a file (**.yml deploy file**) that, in this case, contains:
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]:
```suggestion
The next step is to deploy your container into your AKS Kubernetes cluster. For that you need a file (**.yml deploy file**) that, in this case, contains:
```

**Review by rpetrusha — APPROVED** [review]
I've left a number of suggestions, @mvelosop. You can merge when you're ready.

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 10]
```diff
@@ -3,85 +3,79 @@ title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS
...
+AKS main features are, an Azure-hosted control plane, automated upgrades, self-healing, user configurable scaling, and a simpler user experience for both developers and cluster operators.
 
-- An Azure-hosted control plane
-- Automated upgrades
-- Self-healing
-- User configurable scaling
-- A simpler user experience for both developers and cluster operators.
-
-The following examples explore the creation of an ASP.NET Core 2.1 application that runs on Linux and deploys to an AKS Cluster in Azure, while development is done using Visual Studio 2017.
+In the following examples we'll explore the creation of an ASP.NET Core 2.1 application, using Visual Studio 2017 that runs on Linux and deploys to an AKS Cluster in Azure.
```
[the remark this answers is not in the document]
Comment (mvelosop) [reply docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md]: This paragraph looked familiar to me! @mairaw and I had reviewed this article before and this change was reverting the previous review 😣

**Comment by mvelosop** [comment]
Closing this PR as it was reverting a previous review 😔.

**Pull request closed without merging**